src/sass/app_2/modules/trading.scss

```diff
@@ -258,6 +258,48 @@
                 }
             }
         }
+        &.barriers {
+            .fieldset-header {
+                text-align: center;
+                height: 1.2rem;
+                margin-top: 2px;
+
+                .field-info {
+                    width: 100%;
+                    left: 0 !important;
+                }
+            }
+            .input-field {
+                input {
+                    padding-left: 3px;
+                }
+                &.multiple {
+                    display: inline-block;
+                    width: 46%;
+                    margin-left: 4px;
+                    margin-right: 4px;
+
+                    .up {
+                        mask: url('../images/sidebar/ic_barrier_high.svg');
```

Comment:
Is mask supported widely by other browsers apart from Chromium based ones? Had a look at caniuse.com and it seems most browsers only have partial support.

Reply:
most browsers supported but the -webkit-mask actually, and mask is directly translated into -webkit-mask, but despite that, IE not supported at all. do we need to support both IE and edge, or edge only ? (edge is actually supported)

Reply:
i think its better to make the image into accessible svg and avoid `mask` . i also think thats its quite dangerous in production

Reply:
We can use `mask` if there's some special `clipping` or `filter` effects that we need, but in this case, we're just rendering an `svg` icon. 

So I think `mask` is a tad overkill, especially for something that is only partially supported in most modern browsers.